test/integration/bench/test_bench.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import pytest
+from utils import (get_admin_connection_params, get_log_lines,
+                   run_command_and_get_output)
+from subprocess import Popen, PIPE, STDOUT
+from threading import Thread
+import signal
+import os
+
+def consume_lines(port, pipe):
+    logger = logging.getLogger('localhost:{}'.format(port))
+    with pipe:
+        for line in iter(pipe.readline, b''):
+            logger.warning(line.rstrip().decode('utf-8'))
+
+def test_bench(request, cartridge_cmd, tmpdir):
+
+    base_cmd = [
+        cartridge_cmd, 'bench', '--duration=1'
+    ]
+    command = ["tarantool",
+                """-e box.cfg{listen=3301, work_dir=[[""" + str(tmpdir) +"""]] }""", 
+                """-e box.schema.user.grant("guest", "super", nil, nil, {if_not_exists=true})"""
+    ]
+
+    env = os.environ.copy()
+    process = Popen(command, stdout=PIPE, stderr=STDOUT, env=env)
+    thread = Thread(
+            target=consume_lines,
+            args=["3301", process.stdout]
+        )
+    thread.start()
+    def kill():
+        process.send_signal(signal.SIGKILL)
+        if thread is not None:
+            thread.join(5)
+
+    request.addfinalizer(kill)
+    cmd = base_cmd
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
```

Comment:
Please group the pieces of code (according to logic): use blank lines for this purpose.